LICENSE.md

```diff
@@ -12,6 +12,22 @@ Mapbox Maps for Android version 10.0 ('Mapbox Maps Android SDK') or higher must
 For the full license terms, please see the Mapbox Terms of Service at https://www.mapbox.com/legal/tos/
 
 ## Additional Mapbox Maps Android licenses
+Mapbox Maps Android uses portions of the Activity.
+
+URL: [https://developer.android.com/jetpack/androidx/releases/activity#1.2.3](https://developer.android.com/jetpack/androidx/releases/activity#1.2.3)
+
+License: [The Apache Software License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+===========================================================================
+
+Mapbox Maps Android uses portions of the Activity.
```

Comment:
Hmm why it says `Activity` ? Should be `saved state` I guess

Reply:
looks like the `project` property for both the dependencies gives `Activity` value. this value comes from the license plugin. It should be `saved state` imo as well.
![licence_generate](https://user-images.githubusercontent.com/11589497/152800577-96263caa-241b-4487-964c-5e5d126504d5.png)

Reply:
https://mvnrepository.com/artifact/androidx.savedstate/savedstate indeed, seems the docs are broken